cmd/posix.go

```diff
@@ -41,12 +42,24 @@ const (
 	maxAllowedIOError = 5
 )
 
+const (
+	// readDirentBufSize for syscall.ReadDirent() to hold multiple
+	// directory entries in one buffer. golang source uses 4096 as
+	// buffer size whereas we want 64 times larger to save lots of
+	// entries to avoid multiple syscall.ReadDirent() call.
+	readDirentBufSize = 4096 * 64
+)
+
```

Comment:
since we are using non standard buffer size, is it meaningful to capture this as well along with usage?

Reply:
why? it's not meaningful these are simply for dirents, dirents are not part of disk usage. This buffer contains filesystem direntinfo, it is part of filesystem tree, not the actual usage. We have always used a custom buffer to fetch larger entries in a single readdir operation. 

Disk usage here is simply calculating the total size of the files that are managed by Minio, nothing else.